--- pal_finder_v0.02.04/pal_finder_v0.02.04.pl	2012-03-28 22:17:43.000000000 +0100
+++ pal_finder_v0.02.04.new/pal_finder_v0.02.04.pl	2013-02-26 16:16:47.155247655 +0000
@@ -29,6 +29,8 @@
 
 use strict;
 use warnings;
+# Suppress warnings about deep recursion for large input files
+no warnings 'recursion';
 
 ######################################### DECLARING GLOBAL VARIABLES ###############################################
 
